### PR TITLE
Add Fly.io config

### PR DESCRIPTION
**Pull request by joeriddles**
issue #3

- Add config to deploy app to Fly.io
- Add [dj-django-url](https://pypi.org/project/dj-database-url/) to automatically read `DATABASE_URL` environment variable
- Add basic workflow to deploy to Fly.io on merge to main

~~🛑 blocked on https://github.com/SpokaneTech/SpokaneTech_Py/pull/16 merging~~

### Demo

https://spokanetech.org

image

### CODE REVIEW

**Review by davidslusser — APPROVED** [review]
Dude this looks interesting. I've never used fly.io before, do you have much experience with it? I'm super curious about things like choice of db, support for things like redis/reabbitmq, etc. You should totally give a quick demo of this sometime.

**Comment by joeriddles** [comment]
> Dude this looks interesting. I've never used fly.io before, do you have much experience with it? I'm super curious about things like choice of db, support for things like redis/reabbitmq, etc. You should totally give a quick demo of this sometime.

Yeah I've used it for another Django project. It's pretty slick, kind of similar to Heroku but has a free tier! They've got a cool blog with some Django-focused content: https://fly.io/django-beats/

File: fly.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# fly.toml app configuration file generated for spokanetech-py on 2024-01-31T20:22:30-08:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = 'spokanetech-py'
+primary_region = 'sea'
+console_command = '/code/manage.py shell'
```
Comment (KGB33) [inline fly.toml:8]: I'm not familiar with Fly, but shouldn't this be `/code/src/manage.py`?

Comment (joeriddles) [reply fly.toml:8]: Good call, yes. This will need fixed. I’ll make a PR

**Pull request merged**